docs/next.config.mjs

```diff
@@ -34,7 +34,7 @@ const MONOREPO_PACKAGES = {
 
 export default withDocsInfra(
   /** @type {import('next').NextConfig  }} */ ({
-    transpilePackages: ['@mui/monorepo'],
+    transpilePackages: ['@mui/monorepo', '@mui/x-charts'],
```

Comment:
@Janpot do you know if is there a better way to make this work, or is it fine to transpile `@mui/x-charts` when building the docs?

Reply:
This is fine